spec/models/meeting_spec.rb

```diff
@@ -78,7 +78,7 @@ def make_groups_and_members!(groups, distributions)
   end
   describe 'a perfectly symmetrical example' do
     before :each do
-      groups = %w(      A B C      )
+      groups = %w( A B C      )
```

Comment:
Something's really wrong with rubocop, I tried `rubocop -a`, but it did the same! Might be a bug in rubocop